{
  "FaultLabel": "Resolved power cut",
  "Type": "ResolvedFault",
  "Icon": "icon-success",
  "ShowViewOnMapCTA": false,
  "UnderReview": false,
  "AdditionalFaultInfo": "Good news! The electricity in your area has been restored. The power cut was caused by an unexpected incident on our underground cable that provides your supply.",
  "UpdateInfo": null,
  "CTAs": [],
  "faultNumber": "60514421",
  "multipleFaultNumbers": null,
  "date": "2021-07-15T08:23:45",
  "region": "Poulton-Le-Fylde",
  "faultType": "Resolved Fault",
  "faultStatus": "Resolved",
  "previousFaultStatus": "Resolved",
  "consumersOff": 0,
  "estimatedTimeOfRestoration": "2021-07-15T16:00:00",
  "estimatedTimeOfRestorationMajority": "2021-07-15T08:23:45",
  "information": "",
  "actualTimeOfRestoration": "2021-07-15T08:23:45",
  "outageCentrePoint": {
    "lat": 53.919613,
    "lng": -2.963335
  },
  "outageLocations": null,
  "AffectedPostcodes": " FY6 0EJ, FY6 0HN, FY6 0NP, FY6 0NR, FY6 0NS, FY6 0NT, FY6 0NU",
  "EtrMUnderReview": false,
  "WebTMSFaultType": "U/GM"
}